Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+ToolbarDelegate.swift

```diff
@@ -923,15 +923,19 @@ extension BrowserViewController: UIContextMenuInteractionDelegate {
         })
 
       let pasteAction = UIAction(
-        title: Strings.pasteTitle,
-        image: UIImage(systemName: "doc.on.clipboard"),
         identifier: .paste,
         handler: UIAction.deferredActionHandler { _ in
           if let pasteboardContents = UIPasteboard.general.string {
             // Enter overlay mode and make the search controller appear.
             self.topToolbar.enterOverlayMode(pasteboardContents, pasted: true, search: true)
           }
         })
+      
+      pasteMenuChildren = [pasteGoAction, pasteAction]
+      
+      if isUsingBottomBar {
+        pasteMenuChildren.reverse()
+      }
```

Comment:
This should be manually done for iOS 15, iOS 16 should use the element order API: https://developer.apple.com/documentation/uikit/uicontextmenuconfiguration/elementorder/priority